tests/e2e/cluster_utils.py

```diff
@@ -74,20 +87,34 @@ def get_ols_url(route_name: str) -> str:
         raise Exception("Error getting route hostname") from e
 
 
+def get_pods() -> list[str]:
+    """Get the names of all pods in the cluster."""
+    try:
+        result = run_oc(["get", "pods", "-o", "jsonpath='{.items[*].metadata.name}'"])
+        return result.stdout.strip("'").split()
+    except subprocess.CalledProcessError as e:
+        raise Exception("Error getting pods") from e
+
+
 def get_single_existing_pod_name() -> str:
     """Return name of the single pod that is in the cluster."""
```

Comment:
this should take a NS arg also.  you can default it to `openshift-lightspeed` but it should be passed to oc.